routes/web.php

```diff
@@ -13,22 +13,16 @@
 |
 */
 
-$questions = App\Models\Questions::all();
-$answers = App\Models\Answers::all();
+//$questions = \App\Models\Questions::with('question_votes')
+//    ->orderBy("updated_at", "desc")->get();
```

Comment:
commented code is not allowed only code descriptive and doc comments are allowed